ROMFS/px4fmu_common/init.d/rc.autostart

```diff
@@ -0,0 +1,318 @@
+#
```

Comment:
This file is auto-generated and should not be committed. Please rebase and remove.